ocaml-lsp-server/src/code_actions/diagnostic_util.ml

```diff
@@ -0,0 +1,5 @@
+open Import
+
+let is_unused_variable (d : Diagnostic.t) =
+  String.is_prefix d.message ~prefix:"Error (warning 26): unused variable"
+  || String.is_prefix d.message ~prefix:"Error (warning 27): unused variable"
```

Comment:
1. Naming: `is_unused_var_warning` seems more understandable at once to me
2. Matching on prefix with the error code should be enough, ie `Error (warning 26)`